pom.xml

```diff
@@ -347,8 +347,8 @@
     </properties>
 
     <modules>
-        <module>features/org.wso2.carbon.identity.sso.agent.server.feature</module>
-        <module>components/org.wso2.carbon.identity.sso.tomcat.server</module>
+        <module>features/o.asgardio.tomcat.saml.agent.server.feature</module>
```

Comment:
Typo: o.asgardio.tomcat.saml -> io.asgardio.tomcat.saml

Reply:
Resolved in https://github.com/gayashanbc/identity-agent-sso/pull/1/commits/6f69998146be62873fc13ecd7564b53c08d0f76f